index.html

```diff
@@ -0,0 +1,171 @@
+                <a href=" " ><img src="images/playlist.png"><p>Playlist</p></a>
+                <a href=" " ><img src="images/messages.png"><p>Messages</p></a>
+                <a href=" " ><img src="images/show-more.png"><p>Show more</p></a>
+                <hr>
+            </div>
+            <div class = "subcribed_list">
+                <h3>SUBCRIBED</h3>
+                <a href=" " ><img src="images/Jack.png"><p>Jason Nicho</p></a>
+                <a href=" " ><img src="images/simon.png"><p>Simeon Kris</p></a>
+                <a href=" " ><img src="images/tom.png"><p>Tom Johnson</p></a>
+                <a href=" " ><img src="images/megan.png"><p>Megan Jack</p></a>
+                <a href=" " ><img src="images/cameron.png"><p>Cameron Olivi</p></a>
+                
+            </div>
+          
+        </div>
+          <!-- manin  ----> 
+          <div class="container">
+              <div class="banner">
+                  <img src="images/banner.png">
+              </div>
+              <div class="list_container">
+                  <div class="vid_list">
+                      <a href="play.html"><img src="images/thumbnail1.png" class="thumnail"></a>
+                      <div class="flex-div">
+                          <img src="images/Jack.png">
+                          <div class="vid-info">
+                              <a href="play.html">Learn webcoding in two hours</a>
+                              <p>Jason nichol</p>
+                              <p>20k Views &bull; 2 days ago</p>
+                          </div>
+                      </div>
+                      <div class="vid_list">
+                        <a href=""><img src="images/thumbnail2.png" class="thumnail"></a>
+                        <div class="flex-div">
+                            <img src="images/Jack.png">
+                            <div class="vid-info">
+                                <a href="play.html">Learn webcoding in two hours</a>
+                                <p>Jason nichol</p>
+                                <p>20k Views &bull; 2 days ago</p>
+  
+  
+                            </div>
+                        </div>
+                  </div>
+                  <div class="vid_list">
+                        <a href=""><img src="images/thumbnail3.png" class="thumnail"></a>
+                        <div class="flex-div">
+                            <img src="images/Jack.png">
+                            <div class="vid-info">
+                                <a href="">Learn webcoding in two hours</a>
+                                <p>Jason nichol</p>
+                                <p>20k Views &bull; 2 days ago</p>
+
+
+                            </div>
+                     </div>
+                  </div>
+
+                    <div class="vid_list">
+                        <a href=""><img src="images/thumbnail4.png" class="thumnail"></a>
```

Comment:
I will recommend you remove all empty spaces for the purpose of having a cleaner codes